scripts/nix/flake.nix

```diff
@@ -0,0 +1,47 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+      in {
+        devShells.default = pkgs.mkShell {
+          buildInputs = with pkgs; [
+            bison
+            flex
+            pkg-config
+            gettext
+            cmake
+            python3
+            ghostscript
+            eigen
+            boost
+            glew
+            opencsg
+            cgal
```

Comment:
Note that manifold will soon be the default geometry backend, so it should probably be installed via nix if possible, or we don't have to rely on building it from a submodule.